WinUIGallery/Controls/PageHeader.xaml

```diff
@@ -94,12 +94,49 @@
                     </DropDownButton.Content>
                     <DropDownButton.Flyout>
                         <Flyout Placement="Bottom">
-                            <StackPanel x:Name="SourcePanel" Spacing="4">
-                                <HyperlinkButton x:Name="PageMarkupGitHubLink" ToolTipService.ToolTip="{x:Bind PageMarkupGitHubLink.NavigateUri, Mode=OneWay}">
-                                    <TextBlock Text="XAML source code" />
+                            <StackPanel x:Name="SourcePanel">
+                                <StackPanel x:Name="ControlSourcePanel" Margin="0,0,0,24">
+                                    <StackPanel Orientation="Horizontal" Spacing="8">
+                                        <TextBlock
+                                            Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                                            Style="{StaticResource CaptionTextBlockStyle}"
+                                            Text="Control source code" />
+                                        <FontIcon
+                                            FontSize="14"
```

Comment:
(Just by looking at the number) I think `16` would produce just a bit sharper an icon

Reply:
it becomes a bit out of proportion wrt the text